### ISSUE TITLE
Amazon AWS - build client AMI from desktop client chroot

### ISSUE THREAD

**Issue by imincik** [imincik/gis-lab#405]
Build para-virtualized S3 backed client AMI from desktop client chroot instead of using worker installer scripts (gislab_worker.init.tar.gz)
See:
* https://github.com/brimstone/ami-ubuntu/blob/master/Makefile
* http://blog.hintcafe.com/post/80948702487/create-ami-image-from-scratch-for-ec2-xen
* http://www.technology-mania.com/2011/05/create-ubuntu-ami-from-scratch-on-local.html?m=1
* http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/UserProvidedKernels.html
* https://github.com/tomheady/ec2debian/wiki/64bit-ebs-ami-pvgrub

**Comment by mpastor** [comment]
Need to change in client:
* generate PV-GRUB configuration file /boot/grub/menu.lst with the proper kernel and ramdisk name, e.g.
```
default         0
timeout         2
hiddenmenu

title           GIS.lab OWS worker
root            (hd0)
kernel          /boot/vmlinuz-3.2.0-86-virtual root=/dev/xvda1 init=/sbin/init-gislab ro
initrd          /boot/initrd.img-3.2.0-86-virtual
```
* rename /etc/init/tty1.conf to /etc/init/hvc0.conf and change /dev/tty1 to /dev/hvc0 in this file
* disable server IP chceck in udhcp init script system/roles/client-desktop/templates/initramfs-tools/scripts/init-premount/udhcp.j2  (-I paramter of udhcpc command)
* don't use $ROOTSERVER variable in /etc/hosts section in init file system/roles/client-desktop/files/static/init/gislab-init.sh, because $ROOTSERVER is an IP address of Amazon provided DHCP server. we can move this file to templates and use GISLAB_NETWORK_SERVER_IP ansible variable instead.
* generate DNS recors for entire subnet or use "subnets" in Amazon VPC (more complicated)

**Comment by imincik** [comment]
Implemented in 455fcc7e8157f7c1b37cfa9cbe915ca58591fe49
